### PR TITLE
[eiger] adding Ascent in-situ library for eiger

### PR DESCRIPTION
**Pull request by jfavre**
== COMPLETED: Installation ended successfully (took 1 min 39 secs)
== Results of the build can be found in the log file(s) /scratch/e1000/jfavre/eiger/software/Ascent/0.8.0-cpeGNU-21.12/easybuild/easybuild-Ascent-0.8.0-20220509.092321.log

== COMPLETED: Installation ended successfully (took 1 min 44 secs)
== Results of the build can be found in the log file(s) /scratch/e1000/jfavre/eiger/software/VTK-h/0.8.1-cpeGNU-21.12/easybuild/easybuild-VTK-h-0.8.1-20220509.091824.log

== COMPLETED: Installation ended successfully (took 3 mins 26 secs)
== Results of the build can be found in the log file(s) /scratch/e1000/jfavre/eiger/software/VTK-m/1.7.1-cpeGNU-21.12/easybuild/easybuild-VTK-m-1.7.1-20220509.090818.log
== Build succeeded for 1 out of 1

== COMPLETED: Installation ended successfully (took 2 mins 30 secs)
== Results of the build can be found in the log file(s) /scratch/e1000/jfavre/eiger/software/Conduit/0.8.3-cpeGNU-21.12-python3/easybuild/easybuild-Conduit-0.8.3-20220509.090409.log
== Build succeeded for 3 out of 3

### CODE REVIEW

File: jenkins-builds/1.4.0-21.12-eiger [hunk 1]
```diff
@@ -1,3 +1,4 @@
+ Ascent-0.8.0-cpeGNU-21.12.eb         --set-default-module
```
Comment (lucamar) [inline jenkins-builds/1.4.0-21.12-eiger:1]:
```suggestion
 Ascent-0.8.0-cpeGNU-21.12.eb         --set-default-module
```

Comment (lucamar) [reply jenkins-builds/1.4.0-21.12-eiger:1]: Thanks @jfavre: I would suggest to insert `Ascent-0.8.0-cpeGNU-21.12.eb` together with the group of `cpeGNU` builds in the file `[1.4.0-21.12-eiger](https://github.com/eth-cscs/production/pull/2741/files#diff-e3b814c41bf45b157631cd710087c7b12e06dda89ee4460aa09180593db46b03)`

Comment (jfavre) [reply jenkins-builds/1.4.0-21.12-eiger:1]: ok, the line is there, but at the wrong place? I had assumed we were just using alphabetical order

**Review by jfavre — COMMENTED** [review]
LGTM

**Comment by lucamar** [comment]
I moved `Ascent` to the group of `cpeGNU` builds and removed the `default` tag: there is no need, since it is the only `Ascent` modulefile and Lmod will load it by default.

**Review by lucamar — APPROVED** [review]
Thanks @jfavre

**Pull request merged**